### DISCUSSION TITLE
Implement the custom ner spacy transformer model

### DISCUSSION THREAD

**Discussion by rafeekcse44** [explosion/spaCy#12295] [Help: Coding & Implementations]
Hi Guys,
I am trying to custom ner spacy model.In this mode i am using transformer model(en_core_web_trf). In train data what ever mention only able to get in the result .For example : spring boot mention in train data with skill label.After train the model the result is detect in the format in case of springboot , its not detect.How to improve the model.

**Comment by rmitsch** [comment]
Hi @rafeekcse44, knowing more about your data would help us to give you reasonable advice. Can you share a sample of the data you are training on?
